plugins/node/opentelemetry-instrumentation-mysql2/test/mysql.test.ts

```diff
@@ -154,7 +154,7 @@ describe('mysql2@2.x', () => {
   afterEach(done => {
     context.disable();
     memoryExporter.reset();
-    instrumentation.setConfig();
+    instrumentation.setConfig({});
```

Comment:
Reviewer note: this was due to https://github.com/open-telemetry/opentelemetry-js/pull/4695